source/includes/steps-tutorial-ios-swift.yaml

```
@@ -685,9 +682,9 @@ content: |
          self.present(alertController, animated: true, completion: nil)
      }
 
-  This simply calls the ``logOut`` method on the app we
-  created earlier. In the completion handler, we dispatch to
-  the main thread so that we can navigate back to the welcome screen.
+  This simply calls the ``logOut`` method on the current user of the app we
```

```suggestion
  This calls the ``logOut`` method on the current user of the app we
```

I've heard Nathan lecture about "simply" before (can feel condescending to end users), could we remove this?

I don't feel super strongly about it though so if you think it fits here that's fine too.

---

I agree with you but not for exactly the same reason. In this case, the word applied to some external process rather than to a user action. But it's wordy. Committed your suggestion.